modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItMetaStorageManagerImplTest.java

```diff
@@ -118,7 +118,7 @@ void setUp(TestInfo testInfo, @InjectConfiguration RaftConfiguration raftConfigu
         raftManager.start();
         metaStorageManager.start();
 
-        metaStorageManager.deployWatches();
+        assertThat("Watches were not deployed", metaStorageManager.deployWatches(), willCompleteSuccessfully());
```

Comment:
Please add full stop at the end of the sentence. Related to all such messages.

Reply:
AFAIK that was revised from Ignite 2 times, and now we follow common grammar rules.